4337/docker/bundler/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM docker.io/library/node:18
```

Comment:
The latest LTS is 20. Would it make sense to use the latest version? Or is it the bundler's requirement?

Reply:
It is what they use in their CI: https://github.com/eth-infinitism/bundler/blob/436f12241b06397989fd7a522af0deca0d46ac09/.github/workflows/build.yml#L22

Reply:
> Node.js 18 will be supported until April 2025

Ok we have a plenty of time